validations/src/mainnet_validations.rs

```diff
@@ -12,6 +12,7 @@ const FIRST_EMERGENCY_COMMITTEE: [&str; 7] = [
     "wit1etherz02v4fvqty6jhdawefd0pl33qtevy7s4z",
 ];
 
+const FIRST_HARD_FORK: u32 = "INSERT EPOCH HERE";
```

Comment:
```suggestion
// FIXME(#issuenumber): decide first hard fork epoch
const FIRST_HARD_FORK: u32 = u32::MAX;
```

Reply:
Now, clippy doesn't like "absurd comparisons" :P

Reply:
Good, so we know that it's impossible for `after_first_hard_fork` to return true

Reply:
Joking aside, we can just add `#[allow(clippy::absurd_extreme_comparisons)]`

Reply:
Guys, its OK if clippy does not pass, as this branch is not gonna be merged as it is. Once we have an epoch we just type it there and clippy passes again

Reply:
I thought we wanted to merge this branch and start implementing the breaking changes, and decide on the exact date when we are confident that the implementation is good

Reply:
If it is more helpful for testing purposes I can add it, never thought such a small PR was gonna generate that much discussion haaha